tests/test_log_likelihood.py

```diff
@@ -0,0 +1,88 @@
+"""Assert values log-likelihood function under ambiguity (speed-up)"""
+import os
+from pathlib import Path
+
+import numpy as np
+import pytest
+import respy as rp
+from respy.likelihood import get_crit_func
+from respy.simulate import get_simulate_func
+
+subdir = Path(f"{os.environ['PROJECT_ROOT']}/tests/resources")
```

Comment:
Please full capitalization of module-wide variables.

Reply:
Will be more concise with this in the future.